src/Service/BroadcastsService.php

```diff
@@ -34,4 +36,20 @@ public function findByVersion(
 
         return $this->mapManyEntities($dbEntities);
     }
+
+    public function findByServiceAndDateRange(
+        $serviceId,
```

Comment:
Typehint, this should be a `Sid $serviceId`

Also should have a return typehint of `array`